server/modules/playlist_driver.js

```diff
@@ -20,7 +20,14 @@ const random = require('random-js')();
 const time = require('server/util/time');
 const wallGeometry = require('server/util/wall_geometry');
 
-const driveStateMachine = (playlist, layoutSM) => {
+let timer;
+
+const driveStateMachine = (playlist, layoutSM, clearTimer) => {
+  clearTimer = clearTimer || false;
```

Comment:
because the absense of passing in a var means that variable is undefined, and undefined is falsey, you don't need this line.